core/app/[locale]/(default)/account/(tabs)/addresses/_actions/delete-address.ts

```diff
@@ -53,7 +64,16 @@ export const deleteAddress = async (addressId: number): Promise<State> => {
     revalidatePath('/account/addresses', 'page');
 
     if (result.errors.length === 0) {
-      return { status: 'success', message: t('success') };
+      const updatedData = await getCustomerAddresses({
+        ...(after && { after }),
+        ...(before && { before }),
+      });
+
+      return {
+        status: 'success',
+        message: t('success'),
+        addresses: updatedData?.addresses || [],
+      };
```

Comment:
Why don't we just use Next.js [revalidation](https://nextjs.org/docs/app/api-reference/functions/revalidateTag) + reload the page?

Reply:
@chanceaclark thanks for the suggestion, I simplified the logic and left `revalidatePath` as it is done in adding and editing address. Could you double check the implementation?